odoo/tests/common.py

```diff
@@ -1569,6 +1569,23 @@ def get_http_request_threads():
             self._logger.info('remaining requests')
             odoo.tools.misc.dumpstacks()
 
+    def login_simple(self, username, password):
```

Comment:
Is this really necessary as a toplevel helper method? It seems like something very specific to testing the login flow, most endpoints would just use `authenticate` and go on their merry way.

It's also not clear what's `simple` about it.

Reply:
Thank you for the review!
It has gained some complexity since I first named it. I'll update it and find a new home for it.